vendor/golang.org/x/sys/plan9/mkall.sh

```
@@ -0,0 +1,150 @@
+esac
+
+case "$#" in
+0)
+	;;
+*)
+	echo 'usage: mkall.sh [-n]' 1>&2
+	exit 2
+esac
+
+case "$GOOSARCH" in
+_* | *_ | _)
```

Shellcheck (0.10.0)

[warning] This pattern always overrides a later one on line 111.

(SC2221)